netbox/core/views.py

```diff
@@ -232,3 +237,61 @@ def post(self, request, pk):
         messages.success(request, f"Restored configuration revision #{pk}")
 
         return redirect(candidate_config.get_absolute_url())
+
+#
+# Background Tasks (RQ)
+#
+
+
+class BackgroundQueuesListView(LoginRequiredMixin, View):
+
+    def get(self, request):
+        table = tables.BackgroundQueueTable(get_statistics(run_maintenance_tasks=True)["queues"])
+        table.configure(request)
+        return render(request, 'core/background_tasks.html', {
+            'table': table,
+        })
+
+
+class BackgroundTasksListView(LoginRequiredMixin, View):
+
+    def get(self, request, queue_index):
+        queue_index = int(queue_index)
+        queue = get_queue_by_index(queue_index)
+
+        if queue.count > 0:
+            jobs = queue.get_jobs()
+        else:
+            jobs = []
+
+        table = tables.BackgroundTaskTable(data=jobs, queue_index=queue_index)
+        table.configure(request)
+        return render(request, 'core/background_tasks_queue.html', {
+            'table': table,
+            'queue': queue,
+        })
+
+
+class BackgroundTaskDetailView(LoginRequiredMixin, View):
+
+    def get(self, request, queue_index, job_id):
+        queue_index = int(queue_index)
+        queue = get_queue_by_index(queue_index)
+
+        try:
+            job = RQ_Job.fetch(job_id, connection=queue.connection, serializer=queue.serializer)
+        except NoSuchJobError:
+            raise Http404("Couldn't find job with this ID: %s" % job_id)
```

Comment:
Error message needs to be wrapped with `gettext()`. Could also probably make it shorter, e.g. "Job {ID} not found."